app/views/wizards/claims/add_support_user_wizard/_check_your_answers_step.html.erb

```diff
@@ -0,0 +1,49 @@
+<div class="govuk-width-container">
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+      <%= form_with(model: current_step, url: current_step_path, method: :put) do |f| %>
+
+        <span class="govuk-caption-l"><%= t(".caption") %></span>
+        <h1 class="govuk-heading-l"><%= t(".title") %></h1>
+
+        <%= govuk_summary_list do |summary_list| %>
+          <% summary_list.with_row do |row| %>
+            <% row.with_key(text: User.human_attribute_name(:first_name)) %>
+            <% row.with_value(text: wizard.steps[:support_user].first_name) %>
+            <% row.with_action(text: t(".change"),
+                               href: step_path(:support_user),
+                               visually_hidden_text: User.human_attribute_name(:first_name),
+                               classes: ["govuk-link--no-visited-state"]) %>
+          <% end %>
+
+          <% summary_list.with_row do |row| %>
+            <% row.with_key(text: User.human_attribute_name(:last_name)) %>
+            <% row.with_value(text: wizard.steps[:support_user].last_name) %>
+            <% row.with_action(text: t(".change"),
+                               href: step_path(:support_user),
+                               visually_hidden_text: User.human_attribute_name(:last_name),
+                               classes: ["govuk-link--no-visited-state"]) %>
+          <% end %>
+
+          <% summary_list.with_row do |row| %>
+            <% row.with_key(text: User.human_attribute_name(:email)) %>
+            <% row.with_value(text: wizard.steps[:support_user].email) %>
+            <% row.with_action(text: t(".change"),
+                               href: step_path(:support_user),
+                               visually_hidden_text: User.human_attribute_name(:email),
+                               classes: ["govuk-link--no-visited-state"]) %>
+          <% end %>
+        <% end %>
+
+        <div class="govuk-warning-text">
+          <span class="govuk-warning-text__icon" aria-hidden="true">!</span>
+          <strong class="govuk-warning-text__text">
+            <%= t(".warning", user_name: "#{wizard.steps[:support_user].first_name} #{wizard.steps[:support_user].last_name}") %>
```

Comment:
Again, no biggie. But if you wanted to, you have an initialised Support User object available with a `#full_name`:

```suggestion
            <%= t(".warning", user_name: wizard.steps[:support_user].support_user.full_name) %>
```